src/main/java/org/jitsi/jicofo/BridgeSelector.java

```diff
@@ -251,14 +254,32 @@ synchronized public void removeJvbAddress(String bridgeJid)
      * @param participant the participant for which a bridge is to be selected.
      */
     synchronized public BridgeState selectVideobridge(
-            JitsiMeetConference conference, Participant participant)
+            JitsiMeetConferenceImpl conference, Participant participant)
```

Comment:
Yes. I was planning to remove JitsiConferenceImpl#getJvbJid() (which is the reason Impl is needed here) and JitsiConferenceImpl#getJvbConferenceId(). When I do, I will also remove the usage of Impl.